Cargo.toml

```diff
@@ -174,11 +174,11 @@ ya-sb-router = { git = "https://github.com/golemfactory/ya-service-bus.git", rev
 ya-sb-util = { git = "https://github.com/golemfactory/ya-service-bus.git", rev = "6b494e17d7a662e0b710af8c5a2e99ab4007fdb9"}
 
 ## CLIENT
-ya-client = { git = "https://github.com/golemfactory/ya-client.git", rev = "4099adb99dadbc8929caba33538806f5e10d5920"}
-ya-client-model = { git = "https://github.com/golemfactory/ya-client.git", rev = "4099adb99dadbc8929caba33538806f5e10d5920"}
+ya-client = { git = "https://github.com/golemfactory/ya-client.git", rev = "c3f3419e48eade7f4d491c71948a1a50446d87d5"}
```

Comment:
pls publish `ya-client` as 0.5.1 to crates.io and remove this patch before beta.1

Reply:
i have no idea how :) is there a guide for this?

Reply:
done! :)